mavros_extras/src/plugins/odom.cpp

```diff
@@ -0,0 +1,184 @@
+/**
+ * @brief Odometry plugin
+ * @file odom.cpp
+ * @author James Goppert <james.goppert8@gmail.com>
```

Comment:
@TSC21 should probably add your name here, and also can you fix my email above, has an extra 8.